test/test_vid.py

```
@@ -126,3 +126,39 @@ def test_save_video(tmp_path: Path):
 
     # Clean up the test file
     output_path.unlink()
+
+
+def test_save_video_greyscale(tmp_path: Path):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_save_video_greyscale`

Add return type annotation: `None`

(ANN201)